frontend/public/components/dashboards-page/overview-dashboard/utilization-card.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import * as plugins from '../../../plugins';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { UtilizationItem, UtilizationBody } from '../../dashboard/utilization-card';
+import { DashboardItemProps, withDashboardResources } from '../with-dashboard-resources';
+import { getRangeVectorStats } from '../../graphs/utils';
+import { humanizePercentage, humanizeBinaryBytesWithoutB } from '../../utils';
+import { OverviewQuery, overviewQueries } from './queries';
+
+const getQueries = () => {
+  const pluginQueries = {};
+  const queries = {
```

Comment:
Can we extract this into `public/components/dashboards-page/overview-dashboard/queries.ts` as `utilizationQueries`?

In other words, we should put all queries in one place for easier maintenance and consistency.

Reply:
ok, I put those into `public/components/dashboards-page/overview-dashboard/queries.ts` as `utilizationQueries`